stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -615,16 +618,23 @@ internal class ChannelControllerImpl(
                 var attachment: Attachment = it
                 if (it.upload != null) {
                     val result = uploadAttachment(it, attachmentTransformer)
+                    attachment.extraData[EXTRA_UPLOAD_COMPLETE] = result.isSuccess
```

Comment:
Leo, idea of extension property was using it here as just property, doesn't expose how we use this mutable generic map